pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -486,3 +486,11 @@ func (d *Duration) UnmarshalYAML(unmarshal func(any) error) error {
 		return fmt.Errorf("invalid duration %v", v)
 	}
 }
+
+// swagger:model
+type UpdateRuleGroupResponse struct {
+	Message string   `json:"message,omitempty"`
```

Comment:
NIT: Probably not necessary to `omitempty` the message. We should guarantee at least that to api users.